.github/workflows/commit_check.yaml

```
@@ -3,65 +1,67 @@
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+          cache: 'pip'
+          
+      - name: Run tests
+        run: |
+          pip install pytest pytest-cov
+          pytest --cov=./ --cov-report=xml
+          
+      - name: Upload coverage reports
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)